packs/orion/README.md

```diff
@@ -17,10 +21,48 @@ orion:
 
 This pack uses features released in 1.4+, if your using on an older
 release some changes will be likley (at least removal of the `extra`
-sections in some aliases.
+sections in some aliases and use of `use_none` in rules).
 
 ## Actions
 
-### tbc
+tbc
+
+## Rules
+
+### Start Discovery Webhook
+
+The JSON that should be posted to URL (XXXX) is as follows:
+
+```json
+{'orion_start_discovery': {
```

Comment:
Just a heads up - this doesn't look like a valid JSON to me (strings need to be quoted using double and not single quotes).